### PR TITLE
[Demux] util functions to handle memories

### PR DESCRIPTION
**Pull request by jaeyun-jung**
Update util functions to handle memories in gst-buffer.

### CODE REVIEW

File: gst/nnstreamer/elements/gsttensor_demux.c [hunk 1]
```diff
@@ -511,15 +513,20 @@ gst_tensor_demux_chain (GstPad * pad, GstObject * parent, GstBuffer * buf)
...
       }
       g_strfreev (strv);
       list = list->next;
     } else {
-      mem = gst_buffer_get_memory (buf, i);
-      gst_buffer_append_memory (outbuf, mem);
+      _info =
+          gst_tensors_info_get_nth_info (&tensor_demux->tensors_config.info, i);
+      mem = gst_tensor_buffer_get_nth_memory (buf, i);
+      gst_tensor_buffer_append_memory (outbuf, mem, _info);
```
Comment (songgot) [inline gst/nnstreamer/elements/gsttensor_demux.c:529]: The input argument of each function is checked for null inside the function. 
But what about checking the return value of "gst_tensor_buffer_append_memory" ?
There seem to be many cases where the function returns false.

Comment (songgot) [reply gst/nnstreamer/elements/gsttensor_demux.c:529]: If the value inside info entered in this function is always vaild, it's okay.

**Review by songgot — APPROVED** [review]
LGTM

File: gst/nnstreamer/elements/gsttensor_demux.c (hunk 1 above)
Comment (jaeyun-jung) [reply gst/nnstreamer/elements/gsttensor_demux.c]: Updated, plz see the changes again.

**Review by anyj0527 — APPROVED** [review]
LGTM ✔

**Pull request merged**